kwords.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
+
+def get_keywords(text):
+
+    payload = json.dumps({
+    "text": text,
+    "lang": "en",
+    "n": 10
+    })
+    headers = {
+    'Content-Type': 'application/json'
+    }
+
+
+    # obtain response
+    response = requests.request("POST", url, headers=headers, data=payload)
+
+    keywords, scores, relevant_keywords = response.json()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The response.json() method can raise a ValueError if the response body does not contain valid JSON. This should be handled appropriately.
```diff
  try:
      keywords, scores, relevant_keywords = response.json()
  except ValueError:
      print("Response content is not in JSON format.")
      return None
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->